apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -0,0 +1,487 @@
+import * as React from 'react';
+import {
+  usePopper,
+  createArrowStyles,
+  PositioningProps,
+  PopperVirtualElement,
+  PopperRefHandle,
+} from '@fluentui/react-positioning';
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import { useMergedRefs } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+import { storiesOf } from '@storybook/react';
+import { useFluent } from '@fluentui/react-shared-contexts';
+
+const useStyles = makeStyles({
+  wrapper: {
+    display: 'flex',
+    ...shorthands.gap('5px'),
+    backgroundColor: tokens.colorNeutralBackground1,
+
+    '& .target': {
+      color: tokens.colorNeutralForeground1,
+      backgroundColor: tokens.colorNeutralBackground1,
+      ...shorthands.border('2px', 'dashed', 'green'),
+      width: '400px',
+      height: '200px',
+    },
+  },
+
+  gridWrapper: {
+    width: '400px',
+    height: '400px',
+    display: 'grid',
+    gridTemplateColumns: 'auto auto auto',
+    columnGap: '20px',
+    rowGap: '50px',
+  },
+
+  boundary: {
+    ...shorthands.border('2px', 'dashed', 'red'),
+  },
+
+  box: {
+    ...shorthands.padding('15px'),
+    ...shorthands.border('1px', 'solid', 'blue'),
+    backgroundColor: 'white',
+  },
+
+  arrow: {
+    ...createArrowStyles({ arrowHeight: 5 }),
+    backgroundColor: 'red',
+  },
+
+  seeThrough: {
+    opacity: 0.6,
+  },
+});
+
+const positions = [
+  ['above', 'start'],
+  ['above', 'center'],
+  ['above', 'end'],
+  ['below', 'start'],
+  ['below', 'center'],
+  ['below', 'end'],
+  ['before', 'top'],
+  ['before', 'center'],
+  ['before', 'bottom'],
+  ['after', 'top'],
+  ['after', 'center'],
+  ['after', 'bottom'],
+] as const;
+
+const Box = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>((props, ref) => {
+  const styles = useStyles();
```

Comment:
_Not relevant for this PR_, but it's an interesting idea to use single `makeStyles` in multiple unrelated components. Is this a pattern we recommend applications to use? The downside might be that on the first call of `useStyles` all CSS for all slots is inserted into DOM.

Reply:
well we are flexible right ? 😄 

in all seriousness, I think the use case matters. Here the number of styles is small and performance is not an issue (at least i didn't think it was an issue)

We can obviously very strong handedly recommend to partner teams what to use, but there are other teams within microsoft that we have no control over or and someone will eventually do this